testing/sdk_utils.py

```diff
@@ -177,43 +175,3 @@ def pretty_duration(seconds):
 
 def random_string(length=8):
     return "".join(random.choice(string.ascii_lowercase + string.digits) for _ in range(length))
-
-
-# Pretty much https://github.com/pytoolz/toolz/blob/a8cd0adb5f12ec5b9541d6c2ef5a23072e1b11a3/toolz/dicttoolz.py#L279
-def get_in(keys, coll, default=None):
```

Comment:
As discussed, if we call `toolz.get_in` here and `tools.merge` below, then we would not have to change any of the client code -- but I'm happy to reduce the scope of `sdk_utils` a bit!